bpf/go_common.h

```diff
@@ -80,6 +82,7 @@ struct {
 typedef struct grpc_transports {
     u8 type;
     connection_info_t conn;
+    tp_info_t tp;
```

Comment:
_nit:_ in case we are not type-punning on `type`, perhaps consider adding this above line 83 and moving `type` last to avoid punching a hole in the middle of the object.

Reply:
Good idea